### DISCUSSION TITLE
Jamulus USB stick (JamulusOS like live distribution)

### DISCUSSION THREAD

**Discussion by ann0see** [jamulussoftware/jamulus#2236] [General]
A few days ago I had mail contact with the creator of a new Jamulus OS like distribution: https://www.barbershop.de/de/content/jamulus-os-jamulus-vom-usb-stick

Unfortunately the page is German, but I think many people would be interested in it; especially for bigger music ensambles. Maybe it's worth writing a KB-Entry?

CC: @DetlefHennings

**Comment by henkdegroot** [comment]
Really nice and smart idea to do something like this. Even when you don't speak German you can view the page (as most browsers will be able to translate). After this at least you can get a smile on your face....boy I love these translations. So funny!

Too bad the ISO is also in German language and that it does not work on my system (the OS starts but due to a problem with Jack, Jamulus cannot be used) but guessing that is a hardware problem with the laptop I am using.

_EDIT_: Yes it was an hardware issue, plugged in the USB key into another laptop and it works fine.

**Comment by pcar75** [comment]
I am still using Linux on an USB key for Jamulus session.  Everything takes a long while to boot and settle but once it's done I'm getting from 5 to 17 ms less delay than with Windows 10, depending on the server and internet traffic.

I just hope this 'contact' -if his/her work is published- will not get legal objections, as probably happened to the original JamulusOS developer.

**Comment by klingstar** [comment]
This is a short Jamulus OS manual I wrote in German and deepl.com translated it to English (US). I suppose it is understandable. The software link may change some day. 
[Jamulus OS Kurzanleitung.docx](https://github.com/jamulussoftware/jamulus/files/7875316/Jamulus.OS.Kurzanleitung.docx)
[Jamulus OS manual en.docx](https://github.com/jamulussoftware/jamulus/files/7875317/Jamulus.OS.manual.en.docx)

**Reply by henkdegroot** [reply]
Thank you for these documents, really nice and it contains a link to a more up-to-date version which even contains the latest Jamulus version. I was already trying to get the other iso updated and trying to set it to NL and trying to find how to update jamulus.
Will now continue with this image and see if I need to do anything. Currently waiting for the download to complete and well get it onto my USB shortly after that. Is there an English version as well? Just thinking...if there is an English guide....there might be an English image too.

**Reply by pgScorpio** [reply]
As a matter of fact Jamulus can be installed and run on any Ubuntu live USB stick with persistent storage.

Several tutorials on how to create a bootable Ubuntu live USB stick with persistant storage can be found on the internet.

like [https://www.youtube.com/watch?v=cHF1ByFKtZo](https://www.youtube.com/watch?v=cHF1ByFKtZo)

After that you can just boot into linux and follow the linux install instructions on the Jamulus wiki.

**Reply by henkdegroot** [reply]
Thank you for the hint, I was hoping to have something almost "ready" out of the box, so that I did not have to do all myself. However with this information, I have been able to change the USB image to a Dutch version now. Still strange that the laptop I used previously (which did not work with the audio hardware) is now not booting at all with the USB stick.

**Reply by pgScorpio** [reply]
Yes, a "out of the box" solution that works on all systems would be great, but unfortunately almost impossible.

But all new Linux distro's require Secure UEFI boot to be supported and enabled in the BIOS.
Older PC's/Laptops might not support UEFI boot at all, or need a BIOS upgrade to enable it.
You can also try an older Debian distro, which doesn't use UEFI boot, but those probably won't boot on systems with secure UEFI boot enabled.

Since this is not Jamulus related you'll have to google "UEFI boot" and "Legacy boot" for more information on this topic.

**Reply by henkdegroot** [reply]
> UEFI boot to be supported and enabled in the BIOS

The laptop runs Win 11, so has both enabled.
The laptop which runs Win10, and works with the USB stick also has EUFI enabled but not secure boot.

With "out of the box" I really meant something already prepared/created by someone else which I only have to tweak a little. Some off my band members have occasionally issue with tbeir windows setup, I was investigating if this would be something useful for them.
